_download-meta/openttd-releases.md

```diff
@@ -2,9 +2,13 @@
 folder_old_infrastructure: releases
 ---
 
-For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
-There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
-These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-releases/@@version@@/README.md) for more information.
+OpenTTD requires one version of base graphics; available for installation with the installer are:
```

Comment:
"OpenTTD requires a separate base graphics set to run, which can be downloaded through the installer (Windows) or at first run (Windows, Linux). In case you want to manually download it, you can find it at:"

Reply:
Possibly replace 'download' with 'install'